requirements-test.txt

```diff
@@ -2,4 +2,4 @@ flake8==3.7.7
 nose
 nosexcover
 pep8>=1.6.0,<1.7
-pylint>=1.4.3,<1.5
+pylint
```

Comment:
The pylint pin was updated to track the main st2.git in #10. I'm not sure we're ready to open this up quite yet.